.github/workflows/prod-build.yml

```diff
@@ -385,7 +385,6 @@ jobs:
             --region=$region \
             --source=cloud-function \
             --trigger-http \
-            --allow-unauthenticated \
```

Comment:
I suggest deploying this change to test/staging first to be sure nothing bad will happen?

Also: [This](https://cloud.google.com/functions/docs/securing/managing-access-iam#at_deployment) looks strange:

> Subsequent deployments of the same function don't change its status, even if you don't use this flag.

Reply:
Good catch, I mixed up two options, and we should set `--ingress-settings=internal-and-gclb` in addition to `--allow-unauthenticated`.